src/Defender/Traits/HasRolePermissionsTrait.php

```diff
@@ -19,11 +19,11 @@
      */
     public function permissions()
     {
-        $permissionModel     = config('defender.permission_model');
+        $permissionModel = config('defender.permission_model');
         $permissionRoleTable = config('defender.permission_role_table');
-        $roleKey             = config('defender.role_key');
-        $permissionKey       = config('defender.permission_key');
-        
+        $roleKey = config('defender.role_key');
+        $permissionKey = config('defender.permission_key');
+
```

Comment:
Indeed, but then StyleCI is giving an error on those lines, that's why I fixed them.